en/docs/migrate-to-choreo/api-cloud-vs-choreo-feature-comparison.md

```diff
@@ -0,0 +1,30 @@
+# API Cloud vs Choreo Feature Comparison
+
+| **Feature**                             | **API Cloud**                                                                                                | **Choreo**                                                                                                                                              |
+| ---------------------------------------- | -------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **Environment support**                  | **NO -** A separate organization (tenant) must be created for each environment (development, production, etc). | **YES -** Two environments (development and production)  are supported by default within an organization. Customers can create more environments depending on their requirements. For more details, see [Choreo environments](https://wso2.com/choreo/docs/develop/components/rest-api/#choreo-environments). |
+| **REST APIs**                            | **YES**                                                                                                       | **YES** <br>For more details, see [REST API proxy](https://wso2.com/choreo/docs/develop/components/api-proxy/).                                                                                                                                                |
+| **APIs for SOAP backends**               | **YES**                                                                                                        | Need to implement a REST API from scratch to front the SOAP backend using a preferred programming language.                                             |
```

Comment:
Lets change this to "Can be done by implementing a Ballerina REST API"

Reply:
Updated as suggested.